### ISSUE TITLE
Individual sensor reporting intervals

### ISSUE THREAD

**Issue by zeddy1** [xoseperez/espurna#1102]
Can the sensor reporting intervals be split for each sensor? For example, where a single module has a light sensor and temperature sensor, I'd like the light sensor to report every 30 seconds, whereas the temperature can be reported every 5 minutes.

**Comment by xoseperez** [comment]
You are right, there is no way to do that at the moment. Adding it to the backlog.
